xbmc/utils/StringUtils.cpp

```diff
@@ -1297,7 +1297,7 @@ std::string StringUtils::FormatFileSize(uint64_t bytes)
 {
   const std::array<std::string, 6> units{"B", "kB", "MB", "GB", "TB", "PB"};
   if (bytes < 1000)
-    return Format("%lluB", bytes);
+    return Format("%"PRIu64"B", bytes);
```

Comment:
Do we support a compiler that does not have it? We define PRIu64 ourselves on windows afaik in platformdefs to avoid ugly ifdefs sprinkled around the code